BP/common/culture/cultures/00_east_iranian.txt

```diff
@@ -0,0 +1,795 @@
+			Shura Sidaka Sihya Siladharma Simhyi Skunkha Sthavira Sudarrjam Sudhana Sumauna Sumtharaka Tayasham Tcusyau Vadhaga Vagiti Vaham Vahana
+			Vardhana Varman Vasudeva Vidakai Vidaram Vijita Vijitasimha Vikrama Virsha Visaram Vishirapumna Visma Ysambasta Zambasta
+		}
+		female_names = {
+			Alsamgya Alyonj Armaiti Dharmadasi Khivyaina Maya Nanda Nirmala Praniyata Safali Shandramata Shri Tisyarakshita Vimala Vimalaprabha Yavalac
+		}
+
+		dynasty_title_names = yes
+		founder_named_dynasties = yes
+		
+		dynasty_of_location_prefix = "dynnp_nava"
+		
+		patronym_prefix_male = "dynnpat_pre_puri"
+		patronym_prefix_female = "dynnpat_pre_duda"
+
+		ethnicities = {
+			7 = arab
+			5 = mediterranean
+		}
+	}
+	alan = {
+		graphical_cultures = {
+			byzantine_clothing_gfx
+		}
+		
+		color = { 0.7 0.5 0.7 }
+
+		cadet_dynasty_names = {
+			"dynn_ArdasentA_"
+			"dynn_GA_bA_ratA_"
+			"dynn_DzA_rA_hohtA_"
+			"dynn_DzebysatA_"
+			"dynn_DjerdjitA_"
+			"dynn_GuyrdzybetA_"
+			"dynn_ZA_ndjiatA_"
+			"dynn_KA_rkuystA_"
+			"dynn_DjykkajtA_"
+			"dynn_GodjysatA_"
+			"dynn_PA_rrA_statA_"
+			"dynn_HetA_gkatA_"
+			"dynn_HA_darcatA_"
+			"dynn_TuallA_gtA_"
+			"dynn_FA_rniatA_"
+			"dynn_Ephraimid"
+			"dynn_TandelatA_"
+			"dynn_TsA_ykostA_"
+			"dynn_SozanatA_"
+			"dynn_MA_rzojtA_"
+			"dynn_Ovsia"
+		}
+
+		dynasty_names = {
+			"dynn_ArdasentA_"
+			"dynn_GA_bA_ratA_"
+			"dynn_DzA_rA_hohtA_"
+			"dynn_DzebysatA_"
+			"dynn_DjerdjitA_"
+			"dynn_GuyrdzybetA_"
+			"dynn_ZA_ndjiatA_"
+			"dynn_KA_rkuystA_"
+			"dynn_DjykkajtA_"
+			"dynn_GodjysatA_"
+			"dynn_PA_rrA_statA_"
+			"dynn_HetA_gkatA_"
+			"dynn_HA_darcatA_"
+			"dynn_TuallA_gtA_"
+			"dynn_FA_rniatA_"
+			"dynn_Ephraimid"
+			"dynn_TandelatA_"
+			"dynn_TsA_ykostA_"
+			"dynn_SozanatA_"
+			"dynn_MA_rzojtA_"
+			"dynn_Ovsia"
+		}
+
+		male_names = {
+			E_lbeg E_mbalan Aghsartan Alatey Akhsar Anbal Anbazuk Aravat Arsen Askhkadar Ashot Aslan Aspar Athom Balambar Balsag Barlah BE_qE_tar Basil Bazuk Beler Ber BirE_g Boz-Uruz BuU_rgur Buriberdi
+			Burikhan Candak Chelakhsartag Davit Djadaron Durgulel DzE_rE_s GE_tE_g Georgy Girgon Gokhar Gubaz Guram Hasqar Huddan Istor Itaz Iveri Kandak Kandz Kavtia Kaur
+			Khankhusy KhE_tag Konstantin Kubal Kuddana Kul Kuluk Lak Levan Pulad Respendial Rosmag Safrak Sakhir Sangipan Saros Sathis Savkudz Soslan Suarn Syrdon Taymuraz Taus Totraz Urdure Uruzma Uturku Yasynya Zaur
+		}
+		
+		female_names = {
+			Adadza Alda Altun Ana Atsyrukhs Baia Bedisa Borena Btsenon Burudukan Duda Elene Eliso Gergana Gulchara Guranduht Kera Keratsa Ketevan Koser
+			Lamara Lashin Lela Mariami Natia Nino Rusudan Sanatha Satinik Shatana Shorena Shukhia Tamari Tinatini Tsiala Vardo Vedokha Yeran
+		}	
+		
+		dynasty_of_location_prefix = "dynnp_ve"
+		
+		patronym_suffix_male = "dynnpat_suf_y_fyrt"
+		patronym_suffix_female = "dynnpat_suf_y_chyzg"
+		
+		ethnicities = {
+			7 = arab
+			6 = mediterranean
+		}
+	}
+	khwarezmian = { # WTWSMS
```

Comment:
If this is a culture the mod added you can group it together with the others below, no?

Reply:
All that I changed was the "patronym_prefix_female". Everything else is identical to vanilla like you did for Roman in the Latin culture group.

Reply:
Ok